### PR TITLE
Global search EA

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [x] Yes (**Week of Sep 5**)
- [ ] No

### CODE REVIEW

File: _docs/_user_guide/administrative/access_braze/global_search.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+nav_title: Global Search
+article_title: Global Search
+page_order: 0.5
+page_type: reference
+description: "Learn about global search in Braze."
+---
+
+# Global search
+
+Global search is a powerful feature that helps you locate campaigns, Canvases, segments, and email templates within Braze with greater ease. Use the global search to find what you're looking for without the hassle of scrolling and navigating through menus.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/administrative/access_braze/global_search.md:11]:
```suggestion
Global search is a powerful feature that helps you locate campaigns, Canvases, segments, and email templates within Braze with greater ease. Use the global search to find what you're looking for quickly and easily from anywhere in the Braze platform.
```

File: _docs/_user_guide/administrative/access_braze/global_search.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Each search result is paired with tags that indicate the result's content type (campaign, Canvas, segment, or email template) and status (active, archived, stopped, etc.).
+
+### Access recently opened content
+
+You can easily revisit recently accessed content from the search menu. The search interface displays your recently opened results below the search bar, including items interacted with throughout the entire Braze platform. This lets you quickly return to previously viewed campaigns, Canvases, segments, or email templates so you can pick up right where you left off with fewer clicks.
+
+### Explore more results
+
+If your search returns too many results, scroll to continue viewing more items. As you scroll through the search results dropdown, more items will dynamically load, ensuring you have access to all relevant content.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/administrative/access_braze/global_search.md:64]:
```suggestion

```

File: _docs/_user_guide/administrative/access_braze/global_search.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+page_type: reference
+description: "Learn about global search in Braze."
+---
+
+# Global search
+
+Global search is a powerful feature that helps you locate campaigns, Canvases, segments, and email templates within Braze with greater ease. Use the global search to find what you're looking for quickly and easily from anywhere in the Braze platform.
+
+{% alert important %}
+Global search is currently in early access and is under active development. Please share your thoughts, suggestions, and any issues you encounter using the feedback option at the bottom-right of the search dropdown. If you are interested in participating in the early access, please get in touch with your customer success manager.
```
Comment (lydia-xie) [inline _docs/_user_guide/administrative/access_braze/global_search.md:14]:
```suggestion
Global search is currently in early access and is under active development. Please share your thoughts, suggestions, and any issues you encounter using the feedback option at the bottom-right of the search dropdown. If you are interested in participating in the early access, contact your Braze customer success manager.
```

File: _docs/_user_guide/administrative/access_braze/global_search.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+| Close the search menu       | <kbd>Esc</kbd>  |
+{: .reset-td-br-1 .reset-td-br-2}
+
+### Content type and status tags
+
+Each search result is paired with tags that indicate the result's content type (campaign, Canvas, segment, or email template) and status (active, archived, stopped, etc.).
+
+### Access recently opened content
+
+You can easily revisit recently accessed content from the search menu. The search interface displays your recently opened results below the search bar, including items interacted with throughout the entire Braze platform. This lets you quickly return to previously viewed campaigns, Canvases, segments, or email templates so you can pick up right where you left off with fewer clicks.
```
Comment (lydia-xie) [inline _docs/_user_guide/administrative/access_braze/global_search.md:60]:
```suggestion
You can revisit recently accessed content from the search menu. The search interface displays your recently opened results below the search bar, including items interacted with throughout the entire Braze platform. This lets you return to previously viewed campaigns, Canvases, segments, or email templates so you can pick up right where you left off with fewer clicks.
```

File: _docs/_user_guide/administrative/access_braze/global_search.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+nav_title: Global Search
+article_title: Global Search
+page_order: 0.5
+page_type: reference
+description: "Learn about global search in Braze."
+---
+
+# Global search
+
+Global search is a powerful feature that helps you locate campaigns, Canvases, segments, and email templates within Braze with greater ease. Use the global search to find what you're looking for quickly and easily from anywhere in the Braze platform.
```
Comment (lydia-xie) [inline _docs/_user_guide/administrative/access_braze/global_search.md:11]:
```suggestion
Global search is a powerful feature that helps you locate campaigns, Canvases, segments, and email templates within Braze with greater ease. You can use the global search to find what you're looking for from anywhere in the Braze platform.
```

**Review by lydia-xie — CHANGES_REQUESTED** [review]
looks great! just left some small edits for the words "quickly" and "easily"

**Review by lydia-xie — APPROVED** [review]

**Pull request merged**